### PR TITLE
Update VERSION

### LINKED ISSUE #25: Version needs to be updated for the latest release
The VERSION file has the wrong version.

### PR DESCRIPTION
**Pull request by Hang-Lei-NOAA**
correct version

### CODE REVIEW

**Comment by edwardhartnett** [comment]
Please reference the issue number in all PRs, like this:
Fixes #27

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.6.0
+1.6.2
```
Comment (edwardhartnett) [inline VERSION:1]: What happened to 1.6.1?

**Comment by Hang-Lei-NOAA** [comment]
Thanks, will add it

**Review by BoiVuong-NOAA — APPROVED** [review]

**Comment by Hang-Lei-NOAA** [comment]
I have linked the issue with the PR.

**Comment by edwardhartnett** [comment]
What happened to 1.6.1?

**Comment by Hang-Lei-NOAA** [comment]
To minor development above last version 1.6.0.
1. bug fix
2, documentation added

So, increase 2 on the last digit

**Comment by kgerheiser** [comment]
That's not how it should work. 

If those were separate releases at different times, then sure, but numbers shouldn't be skipped for each feature you add.

**Comment by edwardhartnett** [comment]
OK we will leave that for this release, but that is not how most projects use version numbers.

We increment the final number by 1 for a minor relese.

We increment the middle number by 1 and set final number to 0 for a more major release.

We can discuss on a case by case basis what is major and what is minor.

**Comment by Hang-Lei-NOAA** [comment]
This should be discussed and announced for the new rule.

The tag 1.6.1 should be made several days ago.
for the issue #11 which I opened 24 days ago before documentation is coming.
It was built from old develop and give the version number as 1.6.1. But,
the tag was not made.

Yesterday, when I was asked to add g2c into hpc-stack develop. By the
hpc-stack rule, we do need a tag to enter the develop. So, it is different
from when I asked for tag 1.6.1. So I skipped.

The new rule should include the time for the issue to be addressed and who
should address.

**Comment by edwardhartnett** [comment]
I will address this under NCEPLIBS, since releases will be the same on all NCEPLIBS libraries...

**Pull request merged**